packages/ai-code-mode/tests/create-discovery-tool.test.ts

```
@@ -0,0 +1,63 @@
+import { describe, it, expect } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)